plugins/calibrecompanion.koplugin/main.lua

```diff
@@ -315,11 +315,52 @@ function CalibreCompanion:sendBooklists(arg)
     DEBUG("SEND_BOOKLISTS", arg)
 end
 
+-- TODO: move to more central location
+local lfs = require("libs/libkoreader-lfs")
+
+-- We make the simplifying assumption that no-one will want to store books with
+-- a literal \ in the filename.
+-- FIXME: can calibre-on-windows even send paths with \ in them? If not we don't
+-- need to worry about this.
+local function normalize_path(path)
+    return (path:gsub('\\', '/'):gsub('/+', '/'))
+end
+
+local function path_exists(path)
```

Comment:
Nice to have as a general thing. Could you please add it to util?

https://github.com/koreader/koreader/blob/15ec47d7e019c40013c5440c62862a751e6f8733/frontend/util.lua#L291-L305

Reply:
Moved to `util.pathExists()`